.github/workflows/prod-workflow.yml

```
@@ -193,57 +193,56 @@ jobs:
-  #         SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  fetcher:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packages/fetcher
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          ssh-key: ${{ secrets.PULL_KEY_REPO }}
+      - uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.8)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)